website/guide/api-usage.md

```diff
@@ -11,11 +11,11 @@ For example, you may struggle to:
 * count the number or order of nodes that match a certain pattern
 * compute the replacement string based on the matched nodes
 
-To solve these problems, you can use ast-grep's programmatic API! You can freely inspect and chagne syntax trees in popular programming languages!
+To solve these problems, you can use ast-grep's programmatic API! You can freely inspect and change syntax trees in popular programming languages!
 
 ## Language Bindings
 
-ast-grep supports two programming languages:
+ast-grep provides support for these programming languages:
```

Comment:
At first, I wanted to update it to "three" 😄 Then, decided to use a generic expression here.

Reply:
Thanks! Nice change!